src/main/shortcutHandler.js

```diff
@@ -0,0 +1,295 @@
+import { Menu } from 'electron'
+import path from 'path'
+import isAccelerator from 'electron-is-accelerator'
+import electronLocalshortcut from '@hfelix/electron-localshortcut'
+import { isOsx } from './config'
+import { getKeyboardLanguage } from './keyboardUtils'
+import { isFile, getPath, log, readJson } from './utils'
+
+// Problematic key bindings:
+//   Aidou: Ctrl+/ -> dead key
+//   Inline Code: Ctrl+` -> dead key
+//   Upgrade Heading: Ctrl+= -> points to Ctrl+Plus which is ok; Ctrl+Plus is broken
+
+class Keybindings {
+  constructor () {
+    this.configPath = path.join(getPath('userData'), 'keybindings.json')
+
+    this.keys = new Map([
+      // marktext - macOS only
+      ['mtHide', 'Command+H'],
+      ['mtHideOthers', 'Command+Alt+H'],
+
+      // file menu
+      ['fileNewFile', 'CmdOrCtrl+N'],
+      ['fileNewTab', 'CmdOrCtrl+Shift+T'],
+      ['fileOpenFile', 'CmdOrCtrl+O'],
+      ['fileOpenFolder', 'CmdOrCtrl+Shift+O'],
+      ['fileCloseTab', 'CmdOrCtrl+W'],
+      ['fileSave', 'CmdOrCtrl+S'],
+      ['fileSaveAs', 'CmdOrCtrl+Shift+S'],
+      ['filePrint', 'CmdOrCtrl+P'],
+      ['filePreferences', 'CmdOrCtrl+,'], // marktext menu in macOS
+      ['fileQuit', isOsx ? 'Command+Q' : 'Alt+F4'],
+
+      // edit menu
+      ['editUndo', 'CmdOrCtrl+Z'],
+      ['editRedo', 'CmdOrCtrl+Shift+Z'],
+      ['editCut', 'CmdOrCtrl+X'],
+      ['editCopy', 'CmdOrCtrl+C'],
+      ['editPaste', 'CmdOrCtrl+V'],
+      ['editCopyAsMarkdown', 'CmdOrCtrl+Shift+C'],
+      ['editCopyAsPlaintext', 'CmdOrCtrl+Shift+V'],
+      ['editSelectAll', 'CmdOrCtrl+A'],
+      ['editFind', 'CmdOrCtrl+F'],
+      ['editFindNext', 'CmdOrCtrl+Alt+U'],
+      ['editFindPrevious', 'CmdOrCtrl+Shift+U'],
+      ['editReplace', 'CmdOrCtrl+Alt+F'],
+      ['editAidou', 'CmdOrCtrl+/'],
+
+      // paragraph menu
+      ['paragraphHeading1', 'CmdOrCtrl+1'],
+      ['paragraphHeading2', 'CmdOrCtrl+2'],
+      ['paragraphHeading3', 'CmdOrCtrl+3'],
+      ['paragraphHeading4', 'CmdOrCtrl+4'],
+      ['paragraphHeading5', 'CmdOrCtrl+5'],
+      ['paragraphHeading6', 'CmdOrCtrl+6'],
+      ['paragraphUpgradeHeading', 'CmdOrCtrl+='],
+      ['paragraphDegradeHeading', 'CmdOrCtrl+-'],
+      ['paragraphTable', 'CmdOrCtrl+T'],
+      ['paragraphCodeFences', 'CmdOrCtrl+Alt+C'],
+      ['paragraphQuoteBlock', 'CmdOrCtrl+Alt+Q'],
+      ['paragraphMathBlock', 'CmdOrCtrl+Alt+M'],
+      ['paragraphHtmlBlock', 'CmdOrCtrl+Alt+H'],
```

Comment:
That's strange! I can use `CmdOrCtrl+Alt+H` on dev and production(build version)! but I didn't see the `CmdOrCtrl+Alt+H` in the menu item. the accelerator after the item label is empty in the menu item.

Reply:
<img width="726" alt="2018-11-05 9 51 40" src="https://user-images.githubusercontent.com/9712830/48002016-41ef6080-e145-11e8-90f9-b1e093ac47a8.png">
You can see that the `HTML Block` accelerator is empty.

Reply:
I found the problem: `Command+Alt+H` is defined as hide others (macOS only) and html block. Would you be okay with `Command+Alt+J` on macOS?

Reply:
Ok!